sdk/core/logger/src/log.browser.ts

```diff
@@ -1,7 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-const logFunction = console.debug || console.log;
 export function log(...args: any[]): void {
-  logFunction(...args);
+  if (args.length > 0) {
+    const firstArg = String(args[0]);
+    if (firstArg.includes(":error")) {
+      console.error(...args);
```

Comment:
I'm almost surprised all of these console methods work in IE 11 (I just checked!)

Reply:
I had to look too! 😄